akka-docs-dev/rst/migration-guide-1.0-2.x.rst

```diff
@@ -161,6 +161,23 @@ Update procedure
 
 TODO: code example
 
+`Sink.fanoutPublisher() and Sink.publisher() is now a single method`
+====================================================================
+
+It was a common user mistake to use ``Sink.publisher`` and get into trouble since it would only support
+a single ``Subscriber``, and the discoverability of the apprpriate fix was non-obvious (Sink.fanoutPublisher).
+To make the decision whether to support fanout or not an active one, the aforementioned methods have been
+replaced with a single method: ``Sink.publisher(fanout: Boolean)``.
+
+Update procedure
+----------------
+
+1. Replace all occurences of ``Sink.publisher`` with ``Sink.publisher(false)``
+2. Replace all occurences of ``Sink.fanoutPublisher`` with ``Sink.publisher(true)``
+
+TODO: code example
```

Comment:
@drewhk My point was: In the current release-2.3-dev there are no migration code examples. I suggest you either A) merge this against the current release-2.3-dev branch and then I can create code examples for you to include on your PR so they all go in at once, or merge your PR and I'll rebase on top of that.